src/aimes/emgr/emgr.py

```diff
@@ -309,6 +309,41 @@ def derive_pilot_descriptions(cfg, strategy):
 
 
 # -----------------------------------------------------------------------------
+def derive_pilot_descriptions_osg(cfg, strategy):
+
+    #   Initializing list used to hold pilot descriptions
+    pdescs = list()
+
+    #   Creating the specified number of OSG pilots. Each pilot is given
+    #   the project ID, resource, number of cores, and wall time
+    for pilot in range(cfg['num_osg_pilots']):
+
+        pdesc = rp.ComputePilotDescription()
+
+        pdesc.project = cfg['project_ids'].get('osg.xsede-virt-clust')
+        pdesc.resource = 'osg.xsede-virt-clust'
```

Comment:
By using what is already in place, do you mean for me to parse whether the resource is in the resource (OSG) either supports or not supports (unsupported) OSG?